observability/micrometer/src/main/resources/schema/wildfly-micrometer_1_0.xsd

```diff
@@ -26,17 +26,26 @@
     <!-- The subsystem root element -->
     <xs:element name="subsystem">
         <xs:complexType>
-            <xs:attribute name="security-enabled" type="xs:boolean" default="true">
+            <xs:attribute name="exposed-subsystems" type="xs:string">
                 <xs:annotation>
                     <xs:documentation>
-                        True if authentication is required to access the HTTP endpoint on the HTTP management interface.
+                        The names of the subsystems (separated by spaces) that exposes their metrics in the vendor scope
+                        (or '*' to expose any subsystem metrics).
                     </xs:documentation>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="exposed-subsystems" type="xs:string">
+            <xs:attribute name="endpoint" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation>
+                        The address to where metrics will be published. The OTLP default is http://localhost:4318/v1/metrics,
+                        though WildFly provides no default, so this must be configured manually.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="step" type="xs:string">
                 <xs:annotation>
                     <xs:documentation>
-                        The names of the subsystems (separated by spaces) that exposes their metrics in the vendor scope (or '*' to expose any subsystem metrics).
+                        The step size (reporting frequency) to use in seconds. The default is 60 seconds.
```

Comment:
Also, remove the "The default is 60 seconds." from here. And use XSD to define it.